docs/upgrading.md

```diff
@@ -0,0 +1,67 @@
+# Upgrading
+
+Head Start is a boilerplate meant as a starting point to adjust to a project's needs. It is not a package meant to be used as a dependency. As such, Head Start has no formal releases or versioning.
+
+Did you start your project using Head Start as its template? You can still apply new changes made to Head Start since to your own project. This can be done in 2 steps:
+
+1. Find the change(s) you want to apply
+2. Apply the change(s) to your project
+
+## Find the changes
+
+Since Head Start doesn't have formal versioning, the best changelog is the [list of changes (commits) on the `main` branch](https://github.com/voorhoede/head-start/commits/main/) on GitHub or from the command line in the repository (`git log main --oneline`):
+
+```bash
+6790dba feature: upgrade Astro to v5 beta (#189)
+39298c5 feature: Remove background image when image is loaded (#185)
+822a3a2 test: Link Node, no trailing whitespace (#186)
+3be3ebf fix: redirects middleware (#184)
+23c2ec3 fix: params.locale fallback on [locale]/ route mismatch (#183)
+ae9b5ee chore: component testing (#177)
+83a8b70 feature: LocaleSelector, use localised language names (#182)
+035a205 fix 404: dynamic 404.astro route to render "not found" page  (#180)
+dee8dc3 feat/unit-testing (#169)
+15eb066 Fix: Astro imports in typescript (#176)
+```
+
+Head Start uses a squash-and-merge strategy for pull requests. So the list should read like a clear list of changes. When you've found the change(s) you want to apply to your project, you can continue. The best way to proceed depends on if you want to [apply a single change](#apply-single-change) or [apply a range of changes](#apply-range-of-changes).
+
+## Apply single change
+
+If you only need a single change, you can use a commit's patch file. For example if you select the commit ["feature: upgrade Astro to v5 beta" (`a622bd`)](https://github.com/voorhoede/head-start/commit/a622bd), you can add `.patch` to the URL to get its patch file: [`https://github.com/voorhoede/head-start/commit/a622bd.patch`](https://github.com/voorhoede/head-start/commit/a622bd.patch). Then you can apply the patch to your project from its repository:
+
+```bash
+curl https://github.com/voorhoede/head-start/commit/a622bd.patch | git am
+```
+
```

Comment:
I wouldn't document this option, just the alternative since it's not that much extra work and cuts down on documentation.

Also the codeblock isn't bash I think but should be `shell`. (```shell)